jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/CommonUtils.java

```diff
@@ -66,4 +69,20 @@ public static Long getProcessInstanceId(Command<?> command) {
 
 		return roles;
 	}
+
+	public static Comparator getIgnoreCaseComparator(final QueryContext queryContext){
```

Comment:
@mswiderski I let you weight in here if this is the best place to have this comparator or perhaps move it to its own class.

Reply:
definitely move this to a new class - it does not make much sense to have it as anonymous class